src/components/form/_form.scss

```diff
@@ -175,8 +175,6 @@
     @include reset;
     @include type-style('label-01');
     margin: $spacing-2xs 0 0;
-    max-height: 0;
```

Comment:
just want to point something out for discussion:

currently, this error message is at height 0 and spills outside of the containing `div.bx--form-item`, and relies on the `div.bx--form-item` to have a `margin-bottom` applied to it to achieve proper alignment.

1) if we merge this fix, we will also need to update styles on the web page to fix alignment to remove that 'extra' spacing

2) since we are currently requiring devs using carbon to implement this spacing themselves, just want to verify if we are okay with this somewhat-breaking change (causing visual misalignment that will have to be resolved). i think it slack it came up that we could hold this for v11 onward. but if we think it's okay to push this change for v10, that's cool with me too!

Reply:
1. it sounds like we just need to reduce the amount of margin that we're adding to cover up this overlap bug in the docs right now, is that right?

2. I believe this is alright for v10 since we haven't shipped v10 and so breakages are expected, but I'm interested in other users thoughts on this as well

Reply:
Thanks for pointing this out! I suppose if we're trying to limit the number of breaking changes we introduce with v10 it might be good to hold off, though imo since we are introducing breaking changes already, does it hurt to introduce this change with it?

Reply:
Just to clarify, this change does not affect v9. The overlap bug is present in v10 only so we are actually preventing a breaking change with this PR

Reply:
yah totes imho it should be good to commit this. i wanted to verify if there was a specific reason for the current state, but if no one can justify this 'feature' that leads to visual overlap, then i think we can merge this PR